tsconfig.json

```diff
@@ -15,7 +15,7 @@
       "@shumai/shumai": ["./shumai/index.ts"],
     }
   },
-  "exclude": ["examples/**/*", "build/**/*", "docs/assets/**/*"],
+  "exclude": ["examples/**/*.js", "build/**/*", "docs/assets/**/*"],
```

Comment:
if this is `.js` it breaks the docs for me -- mind changing it back for the time being?

Reply:
No problem! Give me a few as I'm updating the bulleted list in the OG Pr comment to match the order operations are listed in the docs.

Reply:
Done!